### DISCUSSION TITLE
2024년 3월 6일(수) 회의

### DISCUSSION THREAD

**Discussion by Subak-Uncle** [cca-ffodregamdi/running-hi-back-v2#29] [dailyscrum]
# 📋 회의 내용
- 로그인은 카카오 로그인으로 개발 시작 (시큐리티5 미적용으로)
- 3월 말(3월 31일)까지 1차 기능 개발 끝내기.
- 기획 변동 사항 erd diagram에 반영하기
- 다음 주 수요일 까지 팀원 어떤 식으로 구할 지 조사하기
- 백엔드 개발 가능 시간
  - 종완 : 평일 오후 1 시 ~ 5시 (4시간) 
  - 수의 : 평일 오후 1시 ~ 5시 (4시간)
  - 정인 : 월수금 오전 08시 30분 ~ 오후 12시 30분(4시간)
  

# 📌 오늘의 할 일
- [x] 기획 변동 사항에 따른 엔티티 변경
- [x] 피그마 수정
- [x] 좋아요 테이블 연관 관계 삭제(임시)

**Comment by Subak-Uncle** [comment]
병장 김종완! 확인했습니다!

**Comment by silverpoodle** [comment]
확인했슴돠! 저는 엔티티 수정 완료했고 키워드부터 달려보겠습니다!!

**Comment by numerical43** [comment]
모두들 화이팅 화이팅!!
